xbmc/windowing/X11/GLContextEGL.h

```diff
@@ -34,6 +34,7 @@ class CGLContextEGL : public CGLContext
   EGLConfig m_eglConfig;
 protected:
   bool IsSuitableVisual(XVisualInfo *vInfo);
+  bool SuitableCheck(EGLDisplay& eglDisplay, EGLConfig& config);
```

Comment:
```suggestion
  bool SuitableCheck(EGLDisplay eglDisplay, EGLConfig config);
```

Both are pointers, no need for a reference